db/sqlglue.c

```diff
@@ -9012,17 +9012,6 @@ void cancel_sql_statement_with_cnonce(const char *cnonce)
         logmsg(LOGMSG_USER, "Query with cnonce %s not found (finished?)\n", cnonce);
 }
 
-/* log binary cnonce in hex format 
```

Comment:
Good riddance, there probably is a dup for this (eventlog) and possibly elsewhere.